src/components/filters/SearchSettings.tsx

```diff
@@ -34,27 +35,41 @@ const getCurrentSemanticSearchChoice = (queryParams: ParsedUrlQuery) => {
   return exactMatch as string;
 };
 
+const getCurrentPassagesOrderChoice = (queryParams: ParsedUrlQuery) => {
+  return queryParams[QUERY_PARAMS.sort_within_page] === "true";
+};
+
 export const SearchSettings = ({
-  queryParams,
-  handleSortClick,
+  extraClasses = "",
+  handlePassagesClick,
   handleSearchChange,
+  handleSortClick,
+  queryParams,
   setShowOptions,
   settingsButtonRef,
-  extraClasses = "",
 }: TProps) => {
   const searchOptionsRef = useRef(null);
   const [options, setOptions] = useState(sortOptions);
 
+  // no query string OR query string is empty
   const isBrowsing = !queryParams[QUERY_PARAMS.query_string] || queryParams[QUERY_PARAMS.query_string]?.toString().trim() === "";
```

Comment:
note+praise: I've mentioned this before - but "browsing" feels very unclear in code (maybe good for user stories etc) - so nice to have the comment.

Reply:
What might be nice here is a bit of syntactic sugar around query params - a getter function that handles the nuance of "what if there's a param but it has a falsy value".